2-extra/3-groceries.js

```diff
@@ -52,6 +65,11 @@ let numberOfItemsPerWeek = {
   sunday: 0,
 };
 
+for (let aDay in numberOfItemsPerWeek) {
+  console.log(numberOfItemsPerWeek);
```

Comment:
Nitpick: don't leave `console.log` lines in your complete solution unless required by the exercise

Reply:
thanks, changed.